introduction.md

```diff
@@ -0,0 +1,2 @@
+My name is Sam! I'm (trying) to learn the intricacies of git to help out my fellow coworkers when we
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.